mongo/integration/sdam_prose_test.go

```diff
@@ -108,4 +108,34 @@ func TestSDAMProse(t *testing.T) {
 			assert.Soon(t, callback, defaultCallbackTimeout)
 		})
 	})
+
+	mt.RunOpts("client waits between failed Hellos", mtest.NewOptions().MinServerVersion("4.9").Topologies(mtest.Single), func(mt *mtest.T) {
```

Comment:
Only running this test against standalones since it's using `DirectConnection: true` which will error against replica sets and sharded clusters.